gradle.properties

```diff
@@ -7,10 +7,12 @@ minecraft_version=1.13.2
 # Testing: net.minecraftforge.test:forge
 # Normal:  net.minecraftforge:forge
 forge_base=net.minecraftforge:forge
-forge_version=25.0.54
+forge_major=25.0
```

Comment:
Is this really required?

Reply:
For the manifest... Yes - I also forgot to put that into the mods.toml...